.circleci/config.yml

```diff
@@ -729,3 +730,30 @@ jobs:
             cmake --build ./build
             cmake --build ./build --target check-hermes
             python3 hermes/utils/testsuite/run_testsuite.py --test-intl test262/test -b build/bin
+
+  test-linux-test262:
+    docker:
+      - image: debian:bookworm
+    working_directory: /root
+    steps:
+      - checkout:
+          path: hermes
+      - run:
+          name: Setup dependencies
+          command: |
+            apt update
+            apt install -y git openssh-client cmake build-essential \
+                libreadline-dev libicu-dev zip python3
+            # Check out test262 at a pinned revision to reduce flakiness
+            git clone https://github.com/tc39/test262
+            cd test262
+            git checkout 62626e083bd506124aac6c799464d76c2c42851b
+      - run:
+          name: Run test262 with Intl
+          command: |
+            cmake -S hermes -B build -DHERMES_ENABLE_INTL=ON
+            cmake --build ./build
```

Comment:
This is going to be very slow, we should set the parallelism of the build to match the other linux jobs (which pass `-j4`)

Reply:
Ah, good point. Let me edit this.

Reply:
Updated PR to add  `-j 4`.